cmd/run.go

```diff
@@ -209,6 +210,7 @@ updates might cause them to be dropped by OPA.
 	runCommand.Flags().StringVar(&cmdParams.logTimestampFormat, "log-timestamp-format", "", "set log timestamp format (OPA_LOG_TIMESTAMP_FORMAT environment variable)")
 	runCommand.Flags().IntVar(&cmdParams.rt.GracefulShutdownPeriod, "shutdown-grace-period", 10, "set the time (in seconds) that the server will wait to gracefully shut down")
 	runCommand.Flags().IntVar(&cmdParams.rt.ShutdownWaitPeriod, "shutdown-wait-period", 0, "set the time (in seconds) that the server will wait before initiating shutdown")
+	runCommand.Flags().BoolVar(&cmdParams.skipKnownSchemaCheck, "skip-known-schema-check", false, "disables type checking on known input schemas")
```

Comment:
Could `known-schema` be confusing here? To the user, a known schema might be interpreted as one supplied via the `--schema` flag. Perhaps something like `builtin-schema`?

Reply:
It's possible that the "builtin" from `builtin-schema` be interpreted as schema checks on OPA's built-in functions 🤷‍♂️ What about `skip-inbuilt-schema-check` (how creative 😉 ) or `skip-auto-schema-check`? Open to other ideas as well.

Reply:
`predefined-schema`, `inferred-schema`, `recommended-schema`?  `bestest-schema` (😇)

Perhaps there is no perfect name to be found .. and providing an option to opt-out is good enough, and we should just expand the flag description to explain what we mean with "known". 
I definitely won't block the review on this point.

Reply:
I've added the following note in the help text of the `opa run` command.  Let me know if this works.

```
OPA will automatically perform type checking on known input documents and report any errors resulting from the schema
check. Currently this check is performed on OPA's Authorization Policy Input document and will be expanded in the future.
To disable this, use the --skip-known-schema-check flag.
```

Reply:
Yes, this is a good expansion 👍 .
Could we further qualify the text with something like `OPA will automatically perform type checking based on a schema inferred from known input documents ...`?

Reply:
That's nice! Updated the help text.